src/client/src/index.tsx

```diff
@@ -23,7 +23,7 @@ const store = createStoreWithMiddleware(
 );
 
 const vscode = process.env.NODE_ENV === ENVIRONMENT.PRODUCTION ?
-  // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Revert this change to what it was before. 
```suggestion
  // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
```